src/components/settings/SettingsSSOTab/hooks.js

```diff
@@ -72,6 +72,8 @@ const useIdpState = () => {
     formData.append('sync_learner_profile_data', false);
     formData.append('enable_sso_id_verification', true);
 
+    formData.append('debug_mode', providerConfig?.debug_mode || false);
```

Comment:
I'm not seeing `automatic_refresh_enabled` in this list

Reply:
I can add it